### PR TITLE
Added changelog entry

### PR DESCRIPTION
**Pull request by LarryOsterman**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Release History
 
 ## 1.0.0-beta.8 (2024-05-07)
+### Features Added
```
Comment (LarryOsterman) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md:4]:
```suggestion

### Features Added
```

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 ## 1.0.0-beta.8 (2024-05-07)
 
+### Features Added
+
+- Added new features for upcoming Event Hubs service release.
+
+
```
Comment (ahsonkhan) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md:9]:
```suggestion
```

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.8 (2024-05-07)
 
+### Features Added
+
+- Added new features for upcoming Event Hubs service release.
```
Comment (ahsonkhan) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md:7]: This sentence is fairly generic. Do we want to highlight or specify some of the features, so customers know what's new?

For example, something like: Added support for EventHubs emulators.

Comment (LarryOsterman) [reply sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md:7]: This sentence is *intentionally* generic.

**Review by ahsonkhan — APPROVED** [review]

**Pull request merged**